onmt/dynamic/opts.py

```diff
@@ -0,0 +1,139 @@
+"""All options for dynamic running. Should be given in a yaml file."""
+from onmt.opts import _train_general_opts, config_opts, model_opts,\
+    _add_reproducibility_opts
+from onmt.dynamic.transforms import AVAILABLE_TRANSFORMS
+
+
+def _dynamic_corpus_opts(parser, build_vocab_only=False):
+    """Options related to training corpus, type: a list of dictionary."""
+    group = parser.add_argument_group('Data')
+    group.add('-data', '--data', required=True,
+              help='data configuration list.')
+    group.add('-skip_empty_level', '--skip_empty_level', default='warning',
```

Comment:
This one should default to silent I think. A lot of public datasets have a lot of empty lines, and the logs are quite unusable when all the warnings happen.

Reply:
But if no warning at default, the user may not notice any potential failing when the file is erroneous I think.
And those who expect a clean log when using some public datasets can always set this option to `silent`.